src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx

```diff
@@ -93,6 +97,29 @@ export default function ParentMessageInfoItem({
     });
   }, [message?.updatedAt, (message as UserMessage)?.message]);
 
+  // Only for the FileMessageItemBody
+  const downloadFileWithUrl = () => {
+    if (message.messageType === 'file') {
```

Comment:
what about multiple files?

Reply:
We download files on the Fileviewer component for the MFM, not using the link